xarray/namedarray/pycompat.py

```diff
@@ -7,7 +7,8 @@
 import numpy as np
 from packaging.version import Version
 
-from xarray.core.utils import is_duck_array, is_scalar, module_available
+from xarray.core.utils import is_scalar, module_available
+from xarray.namedarray.utils import is_duck_array
```

Comment:
there's a version of `module_availabe` in `namedarray.utils`

```suggestion
from xarray.core.utils import is_scalar
from xarray.namedarray.utils import is_duck_array, module_available
```